src/amo/components/AddonReviewListItem/index.js

```diff
@@ -231,7 +231,14 @@ export class AddonReviewListItemBase extends React.Component<Props> {
         {reviewBody}
         <div className="AddonReviewListItem-byline">
           {review && !isReply ?
-            <Rating styleName="small" rating={review.rating} readOnly />
+            <Rating
+              styleName={
+                siteUser && review && review.userId === siteUser.id ?
+                  'small-by-user' : 'small'
```

Comment:
Nitpick but we shouldn't have any whitespace before the colon here. Not sure if our linter will catch it (but it should 😄)

Reply:
Also, I'd think `own-review` or `my-review` would be a better class name than `small-by-user`. We should decouple ownership `className` from size `className` regardless.